apps/src/templates/teacherDashboard/OwnedSectionsTable.jsx

```diff
@@ -47,7 +47,13 @@ export const sectionLinkFormatter = function (name, {rowData}) {
 };
 
 export const courseLinkFormatter = function (course, {rowData}) {
-  const {assignmentNames, assignmentPaths, courseOfferingsAreLoaded} = rowData;
+  const {
+    assignmentNames,
+    assignmentPaths,
+    courseOfferingsAreLoaded,
+    isAssignedSingleUnitCourse,
+  } = rowData;
+
```

Comment:
this change looks good, but I am just noticing that there is another section table specifically for PL courses that we'll probably want to update: https://github.com/code-dot-org/code-dot-org/blob/58bea267e7ba876b2e2eb3cba67b1b1acc7344aa/apps/src/templates/teacherDashboard/OwnedPlSectionsTable.jsx#L49

 it looks like it will be a bit of work since it will require separate test coverage, so here's a new ticket to track that work: https://codedotorg.atlassian.net/browse/TEACH-1630